src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+    &:nth-child(-n + 4) {
```

Comment:
It should be the same functionality as in the stars task, if a modifier is`stars--3` you should show 3 active stars, if `stars--4` - 4, and so on